src/components/StaticAppBar/StaticAppBar.css

```diff
@@ -19,6 +19,9 @@
 .drawerAppBar button{
     margin-top: 0px !important;
 }
+.top-menu{
+    top: -26px;
```

Comment:
@sudheesh001  since we are using 46ph Height app bar in every screen size this value is not going to be an issue.